rosdep/python.yaml

```diff
@@ -5090,6 +5094,7 @@ python-tk:
   fedora: [python2-tkinter]
   nixos: [pythonPackages.tkinter]
   openembedded: ['${PYTHON_PN}-tkinter@openembedded-core']
+  opensuse: [python-tk]
```

Comment:
Should this one be `python2-tk`, matching the pattern of the rest of the keys?

Reply:
Oh, never mind.  Sorry, just saw the links above.  I guess this one just doesn't fit the pattern :).